AsyncDisplayKit/ASDisplayNodeExtras.h

```diff
@@ -36,7 +36,7 @@ ASDISPLAYNODE_INLINE BOOL ASInterfaceStateIncludesMeasureLayout(ASInterfaceState
   return ((interfaceState & ASInterfaceStateMeasureLayout) == ASInterfaceStateMeasureLayout);
 }
 
-ASDISPLAYNODE_INLINE NSString * _Nonnull NSStringFromASInterfaceState(ASInterfaceState interfaceState)
+__unused static NSString * _Nonnull NSStringFromASInterfaceState(ASInterfaceState interfaceState)
```

Comment:
Now that the assertion failure message in `-[ASDisplayNode dealloc]` has changed, this is unused if `ASDISPLAYNODE_EVENTLOG_ENABLE=0`. `__unused` doesn't hurt anything.